src/main/kotlin/dev/arbjerg/ukulele/command/HelpCommand.kt

```diff
@@ -29,5 +29,20 @@ class HelpCommand : Command("help") {
         addDescription("Displays a list of commands and aliases.")
         addUsage("<command>")
         addDescription("Displays help about a specific command.")
+        addDescription("")
+        addDescription("")
+        addDescription("Here are the currently available commands for Ukulele:")
+        addDescription("loop / l / loopqueue / lq")
+        addDescription("nowplaying / np")
+        addDescription("pause")
+        addDescription("play / p")
+        addDescription("prefix")
+        addDescription("queue / q / list / li")
+        addDescription("repeat / r / repeatone / ro")
+        addDescription("resume")
+        addDescription("say")
+        addDescription("skip / s")
+        addDescription("stop")
+        addDescription("volume / v")
```

Comment:
As a user, I've often wondered what's available at a glance.  This provides that.